src/frontend/screens/AppPasscode/TurnOffPasscode.tsx

```diff
@@ -150,7 +184,7 @@ const ConfirmTurnOffPasswordModal = React.forwardRef<
       enableContentPanningGesture={false}
       enableHandlePanningGesture={false}
       handleHeight={0}
-      handleComponent={() => null}
+      index={-1}
```

Comment:
out of curiosity, is this the change that addresses the flicker? just want that change to be highlighted for future reference :)